Auth0/Authentication.swift

```diff
@@ -251,6 +253,7 @@ public protocol Authentication: Trackable, Loggable {
      - parameter parameters: additional authentication parameters added for Web link. Ignored in other types
 
      - returns: a request
+     - warning: disabled for OIDC-conformant clients, an alternative will be added in a future release
```

Comment:
Add info of what grant type needs to be enabled and a link to the docs about grant type